descriptor/src/scripts/permissionsTable.js

```diff
@@ -0,0 +1,18 @@
+const readYaml = require('../utils/readYaml')
+
+const args = process.argv.slice(2)
+const descriptorFilePath = args[0]
+
+const yaml = readYaml(descriptorFilePath)
```

Comment:
It would be nice to do a bit of sanity checking here to catch that the path exists, and can be read from, and if not, warn the user.